api/src/pcapi/core/users/models.py

```diff
@@ -712,6 +712,12 @@ def has_partner_page(self) -> bool:
         ).scalar()
         return has_partner_page
 
+    @property
+    def is_connect_as(self) -> bool:
+        if self.impersonator:
+            return True
+        return False
```

Comment:
```suggestion
        return bool(self.impersonator)
```
sera plus esthétique :)